cmd/wallet/wallet.go

```diff
@@ -122,7 +122,7 @@ func getFileOrFlag(filename *string, flag *string) (string, error) {
 func init() {
 	inputKDFIterations = WalletCmd.PersistentFlags().Uint("iterations", 2048, "Number of pbkdf2 iterations to perform")
 	inputWords = WalletCmd.PersistentFlags().Int("words", 24, "The number of words to use in the mnemonic")
-	inputAddressesToGenerate = WalletCmd.PersistentFlags().Uint("addresses", 10, "The number of addresses to generate")
+	inputAddressesToGenerate = WalletCmd.PersistentFlags().Uint("addresses", 1, "The number of addresses to generate")
```

Comment:
I don't think we should modify the default here without a good reason. There might be some case where we assume that `polycli wallet generate` would create 10 accounts.